SingularityRunnerBase/src/main/java/com/hubspot/singularity/runner/base/shared/JsonObjectFileHelper.java

```diff
@@ -28,17 +29,21 @@ public JsonObjectFileHelper(ObjectMapper objectMapper) {
 
     log.info("Reading {}", file);
 
-    byte[] bytes = Files.readAllBytes(file);
+    byte[] bytes = new byte[0];
```

Comment:
we get a compile time error if we don't set it at all since we use it in the catch block.